src/views/catalog/customize/CustomizeVirtualMachine.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react';
+import { match } from 'react-router';
+
+import { TemplateParameter, V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { useURLParams } from '@kubevirt-utils/hooks/useURLParams';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  ActionGroup,
+  Button,
+  ExpandableSection,
+  Form,
+  FormGroup,
+  Sidebar,
+  SidebarContent,
+  SidebarPanel,
+  Stack,
+  StackItem,
+  TextInput,
+  Title,
+} from '@patternfly/react-core';
+import ExternalLinkAltIcon from '@patternfly/react-icons/dist/esm/icons/external-link-alt-icon';
+
+import { CustomizeVirtualMachineScheleton } from './components/CustomizeVirtualMachineScheleton';
+
+import '@patternfly/patternfly/patternfly.css';
+import './CustomizeVirtualMachine.scss';
+
+type RightHeadingProps = {
+  template: V1Template;
+};
+
+const RightHeader: React.FC<RightHeadingProps> = ({ template }) => {
```

Comment:
copy the classnames from the link I put above

Reply:
okay thanks